data/transactions/logic/assembler_test.go

```diff
@@ -1029,7 +1113,7 @@ func TestAssembleAsset(t *testing.T) {
 func TestDisassembleSingleOp(t *testing.T) {
 	t.Parallel()
 	// test ensures no double arg_0 entries in disassembly listing
-	sample := "// version 2\narg_0\n"
+	sample := fmt.Sprintf("// version %d\narg_0\n", AssemblerMaxVersion)
```

Comment:
Should these tests run against v2 and v3 instead of just the latest?

Reply:
Yes, thanks.  Adding in cleanup PR.